### ISSUE TITLE
poor quality decode on sequential dropped frames.

### ISSUE THREAD

**Issue by weepy** [xiph/opus#216]
The audio quality of multiple (small) dropped frames is considerably worse than that of a single frame of similar size. 
Is there a reason for this or a way to work around it?

**Comment by weepy** [comment]
E.g. If I decode 4 x 120 dropped frames - it's worse than 1 x 480 frame . 

Is there a reason for this ?

**Comment by xnorpx** [comment]
I was expecting some worse PLC but this looks like a bug to me
@jmvalin @mark4o might have more insight.

Repro:
Reference (no loss)
Encode decode 24% packet loss single packet 10ms frames 48kHz audio mode 256000 bitrate
lost = (((count + 1) % 4) == 0) ? 1 : 0;
opus_demo audio 48000 1 256000 -framesize 10 five.pcm out.pcm
Encode decode 24% packet loss 4 packet burst loss 2.5ms frames 48kHz audio mode 256000 bitrate
lost = (((count + 4) % 16) < 4) ? 1 : 0;
opus_demo audio 48000 1 256000 -framesize 2.5 five.pcm out.pcm

![image](https://user-images.githubusercontent.com/302709/106103350-a9fb3a00-60f5-11eb-896f-ff5e68620901.png)

**Comment by weepy** [comment]
Should you also set  OPUS_SET_PACKET_LOSS_PERC ?

**Comment by xnorpx** [comment]
Right, that's true the setting control packet dependency so it makes sense. Can you share samples what you hear and the exact settings of the quality issue you experience?

![image](https://user-images.githubusercontent.com/302709/106186775-92ee3380-6159-11eb-8fbc-664d36592fbb.png)
